<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddCamera</class>
 <widget class="QDialog" name="AddCamera">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>743</width>
    <height>250</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>438</width>
    <height>180</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>210</y>
     <width>231</width>
     <height>32</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>10</y>
     <width>101</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="NameText">
      <property name="text">
       <string>Name:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="Name">
      <property name="text">
       <string>Camera</string>
      </property>
      <property name="maxLength">
       <number>16</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="zorn">
      <property name="text">
       <string>Diagonal angle</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="zorny_uhol">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="Play_ID">
   <property name="geometry">
    <rect>
     <x>414</x>
     <y>28</y>
     <width>28</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>Pictures/show_camera.ico</normaloff>Pictures/show_camera.ico</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="deviceUSB_ID">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>30</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="device">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Device ID:</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>10</y>
     <width>151</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>131</width>
      <height>110</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="Position">
       <property name="text">
        <string>Position:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="X">
       <property name="inputMethodHints">
        <set>Qt::ImhDigitsOnly</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Y">
       <property name="inputMethodHints">
        <set>Qt::ImhDigitsOnly</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Z">
       <property name="inputMethodHints">
        <set>Qt::ImhDigitsOnly</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>16</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="x">
       <property name="text">
        <string>x:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="y">
       <property name="text">
        <string>y:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="z">
       <property name="text">
        <string>z:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>60</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLabel" name="FrameC">
      <property name="text">
       <string>Frame Cols</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="FrameCols"/>
    </item>
    <item>
     <widget class="QLabel" name="FrameR">
      <property name="text">
       <string>Frame Rows</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="FrameRows"/>
    </item>
   </layout>
  </widget>
  <widget class="QCheckBox" name="useBackgroundSub">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>140</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Use BackgroudSubstractor</string>
   </property>
  </widget>
  <widget class="QLabel" name="warning">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>205</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>232</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="BrightText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>254</red>
         <green>246</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>252</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ToolTipText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>232</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="BrightText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>254</red>
         <green>246</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>252</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ToolTipText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>190</red>
         <green>190</green>
         <blue>190</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>190</red>
         <green>190</green>
         <blue>190</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="BrightText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>254</red>
         <green>246</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>190</red>
         <green>190</green>
         <blue>190</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ToolTipText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="CQtOpenCVViewerGl" name="glImage" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>320</width>
     <height>240</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>180</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Undistort calibration</string>
   </property>
  </widget>
  <widget class="QPushButton" name="readYAML">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>210</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>ReadYAML</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CQtOpenCVViewerGl</class>
   <extends>QWidget</extends>
   <header>Gui/camwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddCamera</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Play_ID</sender>
   <signal>clicked(bool)</signal>
   <receiver>deviceUSB_ID</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>220</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
